Documentation/admin/install-multi.md

```diff
@@ -0,0 +1,205 @@
+<br>
+<div class="alert alert-info" role="alert">
+<i class="fa fa-exclamation-triangle"></i><b> Note:</b> This documentation is for an alpha feature. For questions and feedback on the Multi-cluster Alpha program, email <a href="mailto:tectonic-alpha-feedback@coreos.com">tectonic-alpha-feedback@coreos.com</a>.
+</div>
+
+# Installing the multi-cluster registry
+
+Use Tectonic's multi-cluster registry to:
+* maintain a list of available clusters
+* enable cluster administrators to configure base security policies, and enforce other standardization policies across multiple clusters
+
+Use this guide to install and configure the registry.
+
+## Install the registry
+
+Install the multi-cluster registry by downloading and applying the following manifests:
+* [cluster-crd.yaml][cluster-crd]
```

Comment:
nit: can we concatenate these together so they're just in one file? It's easier to kubectl apply e.g. something like https://gist.github.com/ericchiang/6d181e119fcbd00d1ed6dbe6faebef47